checkmackerelmetric/checkmackerelmetric.go

```diff
@@ -85,7 +87,17 @@ func (opts *mackerelMetricOpts) run() *checkers.Checker {
 
 	conf, err := config.LoadConfig(config.DefaultConfig.Conffile)
 	if err != nil {
-		return checkers.Unknown(fmt.Sprintf("%v", err))
+		if runtime.GOOS == "windows" {
+			newpath := filepath.Join(config.DefaultConfig.Conffile, "../../../mackerel-agent.conf")
```

Comment:
[note]
It will probably be something like this:
```
C:\Program Files\Mackerel\mackerel-agent\plugins\bin\mackerel-agent.conf\..\..\..\mackerel-agent.conf
 ↓
C:\Program Files\Mackerel\mackerel-agent\mackerel-agent.conf
```

Reply:
yes! it looks ugly workaround, but it is hard to change mackerel-agent config logic.